src/test/java/seedu/address/storage/JsonAdaptedRecipeTest.java

```diff
@@ -69,8 +70,10 @@ public void toModelType_nullTime_throwsIllegalValueException() {
 
     @Test
     public void toModelType_invalidStep_throwsIllegalValueException() {
+        List<JsonAdaptedStep> invalidSteps = new ArrayList<>(VALID_STEP);
+        invalidSteps.add(new JsonAdaptedStep(INVALID_STEP));
         JsonAdaptedRecipe recipe =
-                new JsonAdaptedRecipe(VALID_NAME, VALID_TIME, VALID_INGREDIENTS, INVALID_STEP, VALID_GOALS);
+                new JsonAdaptedRecipe(VALID_NAME, VALID_TIME, VALID_INGREDIENTS, invalidSteps, VALID_GOALS);
```

Comment:
It might be better to declare the invalidSteps above as a private static final INVALID_STEP variable!